.github/workflows/ci.yml

```diff
@@ -130,8 +130,8 @@ jobs:
     name: Pytest Ubuntu
     strategy:
       matrix:
-        # TODO(#3800): remove 3.6 when Colab switches to 3.7
-        python-version: [ '3.6', '3.7', '3.8' ]
+        # TODO(#3800): remove 3.6 when Colab switches to 3.8
+        python-version: [ '3.6', '3.7', '3.8', '3.9' ]
```

Comment:
This might have to be a separate PR, with potentially a bunch of issues see #3772.

Reply:
Yes, I agree that having it separate would be much better 👍